### PR TITLE
Running cells

### PR DESCRIPTION
**Pull request by hbcarlos**
Reviewed nulls and add execution method to run cell at the initialisation.

### CODE REVIEW

File: src/editor/gridstack/gridstackWidget.ts [hunk 1]
```diff
@@ -253,10 +244,10 @@ export class GridstackWidget extends Widget {
     items.forEach(el => {
       this._model.setCellInfo(el.id as string, {
         hidden: false,
-        col: el.x!,
-        row: el.y!,
-        width: el.width!,
-        height: el.height!
+        col: el.x || 0,
```
Comment (jtpio) [inline src/editor/gridstack/gridstackWidget.ts:247]: I guess this could also be written with the `??` operator?

Comment (jtpio) [reply src/editor/gridstack/gridstackWidget.ts:247]: Even though in this case it should be pretty similar since these fields are numbers.

Comment (hbcarlos) [reply src/editor/gridstack/gridstackWidget.ts:247]: Doing a commit

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # jupyterlab-gridstack
 
-![Github Actions Status](https://github.com/hbcarlos/jupyterlab-gridstack/workflows/Build/badge.svg)[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/hbcarlos/jupyterlab-gridstack/master?urlpath=lab)
+![Github Actions Status](https://github.com/hbcarlos/voila-editor/workflows/Build/badge.svg)[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/hbcarlos/voila-edito/master?urlpath=lab)
```
Comment (jtpio) [inline README.md:3]:
```suggestion
![Github Actions Status](https://github.com/hbcarlos/voila-editor/workflows/Build/badge.svg)[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/hbcarlos/voila-editor/master?urlpath=lab)
```

Comment (jtpio) [reply README.md:3]: Missing 'r'

File: src/editor/gridstack/gridstackWidget.ts (hunk 1 above)
Comment (jtpio) [reply src/editor/gridstack/gridstackWidget.ts]: :+1:

**Comment by jtpio** [comment]
Thanks!

**Pull request merged**